clang/lib/Parse/ParseDecl.cpp

```diff
@@ -7986,6 +7986,13 @@ void Parser::ParseParameterDeclarationClause(
     if (getLangOpts().HLSL)
       MaybeParseHLSLAnnotations(DS.getAttributes());
 
+    if (ParmDeclarator.getIdentifier() == nullptr &&
```

Comment:
I don't think checking checking for an identifier makes sense, the type is enough

Reply:
@cor3ntin thanks for the feedback. I've removed it.